.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
```

Comment:
are to start including a changeset for releases or?

Reply:
Seems like the convention is to keep this file after looking at other big name projects that use it, e.g. https://github.com/emotion-js/emotion/tree/main/.changeset